modules/virt-preventing-workload-updates-during-eus-update.adoc

```diff
@@ -0,0 +1,164 @@
+
+Before you begin to update from one Extended Update Support (EUS) version to the next, you must manually disable automatic workload updates.
+
+.Prerequisites
+
+* You are running an EUS version of {product-title} and want to update to the next EUS version. You have not yet updated to the odd-numbered version in between.
+
+* You read "Preparing to perform an EUS-to-EUS update" and learned the caveats and requirements that pertain to your {product-title} cluster.
+
+* You paused the worker nodes' machine config pools as directed by the {product-title} documentation.
+
+.Procedure
+
+. Back up the current `workloadUpdateMethods` configuration by running the following command:
++
+[source,terminal]
+----
+$ WORKLOAD_UPDATE_METHODS=$(oc get kv kubevirt-kubevirt-hyperconverged -n openshift-cnv -o jsonpath='{.spec.workloadUpdateStrategy.workloadUpdateMethods}')
+----
+
+. Turn off all workload update methods by running the following command:
++
+[source,terminal]
+----
+$ oc patch hco kubevirt-hyperconverged -n openshift-cnv --type json -p '[{"op":"replace","path":"/spec/workloadUpdateStrategy/workloadUpdateMethods", "value":[]}]'
+----
++
+.Example output
+[source,terminal]
+----
+hyperconverged.hco.kubevirt.io/kubevirt-hyperconverged patched
+----
+
+. Ensure that the `HyperConverged` Operator is `Upgradeable` before you continue. Enter the following command and monitor the output:
++
+[source,terminal]
+----
+$ kubectl get hco kubevirt-hyperconverged -n openshift-cnv -o json | jq ".status.conditions"
+----
++
+.Example output
+[%collapsible]
+====
+[source,json]
+----
+[
+  {
+    "lastTransitionTime": "2022-12-09T16:29:11Z",
+    "message": "Reconcile completed successfully",
+    "observedGeneration": 3,
+    "reason": "ReconcileCompleted",
+    "status": "True",
+    "type": "ReconcileComplete"
+  },
+  {
+    "lastTransitionTime": "2022-12-09T20:30:10Z",
+    "message": "Reconcile completed successfully",
+    "observedGeneration": 3,
+    "reason": "ReconcileCompleted",
+    "status": "True",
+    "type": "Available"
+  },
+  {
+    "lastTransitionTime": "2022-12-09T20:30:10Z",
+    "message": "Reconcile completed successfully",
+    "observedGeneration": 3,
+    "reason": "ReconcileCompleted",
+    "status": "False",
+    "type": "Progressing"
+  },
+  {
+    "lastTransitionTime": "2022-12-09T16:39:11Z",
+    "message": "Reconcile completed successfully",
+    "observedGeneration": 3,
+    "reason": "ReconcileCompleted",
+    "status": "False",
+    "type": "Degraded"
+  },
+  {
+    "lastTransitionTime": "2022-12-09T20:30:10Z",
+    "message": "Reconcile completed successfully",
+    "observedGeneration": 3,
+    "reason": "ReconcileCompleted",
+    "status": "True",
+    "type": "Upgradeable"
+  }
+]
+----
+====
+
+. Manually update your cluster from the source EUS version to the next minor version of {product-title}:
++
+[source,terminal]
++
+====
+$ oc adm upgrade
+====
++
+.Verification
```

Comment:
Should this be a `.Verification` in the middle of a procedure? Looks a bit odd to me.